.github/template-featureRequest.md

```diff
@@ -1,23 +1,18 @@
-**Feature / Enhancement Request**
+**Issue Title:** Feature: [your request in a line] OR Enhancement: [your suggestion in a line]
 
-**Issue Title:** Feature Request: [your request in a line]
+Copy-and-paste the following content to the issue tracker and fill in with relevant details:
+```markdown
+**Describe the feature/enhancement you would like to see added to TEAMMATES.**
 
-**Proposed Feature / Enhancement**
 
-> Give us a brief description of the feature / enhancement you would like to see added to TEAMMATES,
->
-> e.g. A mobile web app for TEAMMATES
 
-**Problem it solves**
+**How would your proposed feature/enhancement benefit TEAMMATES?**
 
-> Tell us how your proposed feature / enhancement will benefit TEAMMATES,
->
-> e.g. Many students use TEAMMATES on their phones and the current mobile version is not ideal because...
 
-**Developer/User-facing**
 
-> Let us know if the proposed feature is for the benefit of the developer or user
+**Are there existing similar features? If yes, why do they fall short of your expectations?**
 
-**Existing similar features (if any), and their shortcomings**
 
-> Let us know if there already are similar features in TEAMMATES, and why they fall short of your expectations
+
+**Is the proposed feature/enhancement for the benefit of developers or users?**
```

Comment:
"Will the proposed feature/enhancement benefit the developers or users?" sounds better.